rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/ldap/LdapClient.java

```diff
@@ -232,13 +232,14 @@ public AccessIdRepresentationModel searchAccessIdByDn(final String dn) {
     // given DN.
     // https://stackoverflow.com/questions/55285743/spring-ldaptemplate-how-to-lookup-fully-qualified-dn-with-configured-base-dn
     // Therefore we have to remove the base name from the dn before performing the lookup
-    String nameWithoutBaseDn = getNameWithoutBaseDn(dn);
+    String formattedName = getNameWithoutBaseDn(dn).toLowerCase();
```

Comment:
I don't like the name `formattedName`. 
Why did you change it anyway? `nameWithoutBaseDn` was already fine.
If you want to include that the name is lowercase: `lowerCaseNameWithoutBaseDn`?

Reply:
done.